helm/appcatalog-chart/templates/appcatalog.yaml

```
@@ -3,6 +3,7 @@ kind: AppCatalog
 metadata:
   name: "{{ .Values.appCatalog.name }}"
   labels:
+    application.giantswarm.io/catalog-type: "{{ .Values.appCatalog.labels.catalogType }}"
```

```suggestion
    application.giantswarm.io/catalog-type: "{{ .Values.appCatalog.catalogType }}"
```

Not sure about the labels in the path. As this is just the value of label. WDYT?